jibri/rootfs/defaults/jibri.conf

```diff
@@ -118,6 +123,20 @@ jibri {
       audio-source = "pulse"
       // The audio device that will be used to capture audio on Linux
       audio-device =  "default"
+      // Available presets: ultrafast, superfast, veryfast, faster, fast, medium,
+      // slow, slower, veryslow, placebo
+      video-encode-preset = "{{ $JIBRI_RECORDING_VIDEO_ENCODE_PRESET }}"
+      // The range of the CRF scale is 0-51, where 0 is lossless,
+      // 23 is the default, and 51 is worst quality possible.
+      h264-constant-rate-factor = "{{ $JIBRI_RECORDING_CONSTANT_RATE_FACTOR }}"
+      // Settings based upon the specifics of your input. Current tunings include:
+      // film, animation, grain, stillimage, fastdecode, zerolatency, psnr, ssim
+      tune = "{{ $JIBRI_RECORDING_TUNE }}"
```

Comment:
I don't see this being configurable.

Reply:
Are you saying that I can only consider options defined in [this](https://github.com/jitsi/jibri/blob/master/src/main/kotlin/org/jitsi/jibri/capture/ffmpeg/FfmpegCapturer.kt) file?

Or are you saying that _tune_ is not configurable in **ffmpeg** (which is wrong IMHO and up the [documentation](https://trac.ffmpeg.org/wiki/Encode/H.264#a2.Chooseapresetandtune))?

Reply:
The former.

Reply:
I replaced the non-configurable ones with `JIBRI_RECORDING_FRAMERATE`, `JIBRI_RECORDING_QUEUE_SIZE` and `JIBRI_RECORDING_STREAMING_MAX_BITRATE`.